### PR TITLE
Build release with actions

### PR DESCRIPTION
**Pull request by philkohl**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -132,49 +132,61 @@ jobs:
...
+        run: |
+          # Install gpg secret key
+          cat <(echo -e "${{ secrets.OSSRH_GPG_KEY }}") | gpg --batch --import
+          # Verify gpg secret key
+          gpg --list-secret-keys --keyid-format LONG
+      - name: Deploy with Maven
+        env:
+          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+        run: mvn deploy -ntp -Dgpg.passphrase=${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD_NALUKIT_GITHUB_USER }} -DskipTests -Daether.checksums.algorithms=MD5
```
Comment (FrankHossfeld) [inline .github/workflows/build.yaml:192]: hier fehlt das Profil release : -P release

File: .github/workflows/build.yaml
Comment (FrankHossfeld) [inline .github/workflows/build.yaml:1]: Das hier:
 - name: Upload OWASP results
        if: always()
        uses: actions/upload-artifact@master
        with:
          name: OWASP report
          path: ${{github.workspace}}/reports

sollte nach:

       - name: Upload OWASP results
        if: always()
        uses: actions/upload-artifact@master
        with:
          name: OWASP report
          path: ${{github.workspace}}/**/reports/dependency-check-report.html

geändert werden

**Review by FrankHossfeld — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (philkohl) [reply .github/workflows/build.yaml]: done

File: .github/workflows/build.yaml
Comment (philkohl) [reply .github/workflows/build.yaml:1]: done

**Review by FrankHossfeld — APPROVED** [review]

**Comment by FrankHossfeld** [comment]
This PR will solve https://github.com/NaluKit/nalu/pull/311

**Pull request merged**